bower.json

```diff
@@ -7,9 +7,15 @@
     "jquery.cookie": "~1.4.1",
     "bootstrap": "~3.3.1",
     "datatables-bootstrap3-plugin": "~0.2.0",
-    "angular": "~1.3.13",
+    "angular": "~1.3.15",
+    "angular-csv-import": "~0.0.15",
+    "angular-loading-bar": "~1.3.15",
     "angular-ui-router": "~0.2.13",
+    "angular-ui-select": "~0.11.2",
+    "angular-ui-tree": "~2.2.0",
     "angular-gettext": "~2.0.2",
+    "angular-sanitize": "~1.3.15",
+    "angular-xeditable": "~0.1.9",
```

Comment:
I checked the licenses and the last release dates of the new angular modules:
 * angular-csv-import: MIT, last release 8.4.2015
 * angular-loading-bar: MIT, last release 24.2.2015
 * angular-ui-select: MIT, last release: 17.3.2015
 * angular-ui-tree: MIT, last release: 30.1.2015
 * angular-sanitize: MIT (part of angularjs)
 * angular-xeditable: MIT, last release: 26.3.2015

Conclusion: Yes, all packages are Free Software and active developed.